src/bucket/BucketManagerImpl.cpp

```diff
@@ -188,6 +188,21 @@ BucketManagerImpl::BucketManagerImpl(Application& app)
     , mDeleteEntireBucketDirInDtor(
           app.getConfig().isInMemoryModeWithoutMinimalDB())
 {
+    for (uint32_t t =
```

Comment:
Nit: An old C style trick is to add a last place holder value in an enum for easier, less footgunny iteration. See `enum TestDbMode` for an example, but the last element is usually called something like `NUM_TYPES`.